File: 30 Days of Code/Day 2: Operators/solution.pas
var meal_cost:real; 
    tip_percent, tax_percent:integer;
begin
    readln(meal_cost);
    readln(tip_percent);
    readln(tax_percent);
    write(round(meal_cost * tip_percent/100 + meal_cost * tax_percent/100 + meal_cost));
    readln
end.

File: 30 Days of Code/Day 2: Operators/solution.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    double meal_cost;
    int tip_percent, tax_percent;
    cin >> meal_cost;
    cin >> tip_percent;
    cin >> tax_percent;
    cout<<round(meal_cost * tip_percent/100+ meal_cost * tax_percent/100+meal_cost);
    return 0;
}
